OpenChange/MAPIStoreMessage.m

```diff
@@ -178,56 +178,109 @@ - (NSDictionary *) _convertRecipient: (struct mapistore_message_recipient *) rec
                           andColumns: (struct SPropTagArray *) columns
 {
   NSMutableDictionary *recipientProperties;
+  enum MAPITAGS prop_tag;
+  char *displayName = NULL;
+  char *email = NULL;
+  uint8_t object_type = 0;
+  char *smtpAddress = NULL;
   SOGoUser *recipientUser;
   NSUInteger count;
+  id recipientEmail = nil;
+  id recipientFullName = nil;
```

Comment:
@javierag Why do you set `recipientEmail` and `recipientFullName` as `id` when it was correctly set to `NSString *`? The one without modification must be `value` which is generic as stores different kind of pointers.